src/components/KPagination/KPagination.vue

```diff
@@ -131,12 +132,13 @@
 
 <script setup lang="ts">
 import type { Ref, PropType } from 'vue'
-import { ref, computed, watch } from 'vue'
+import { ref, computed, watch, onMounted, onUnmounted, nextTick } from 'vue'
 import KDropdown from '@/components/KDropdown/KDropdown.vue'
 import KButton from '@/components/KButton/KButton.vue'
 import PaginationOffset from './PaginationOffset.vue'
 import type { PageSizeChangeData, PageChangeData, DropdownItem } from '@/types'
 import { BackIcon, ForwardIcon, ChevronDownIcon } from '@kong/icons'
+import { ResizeObserverHelper } from '@/utilities/resizeObserverHelper'
```

Comment:
(I realize this is still a draft)
If we’re going to expand the usage of this resize observer helper, I think we first need to debug the errors it seems to be throwing in Datadog on a regular basis

Reply:
Is it a blocker for this PR? I remember Kai mentioning something he wanted to try that could solve the ResizeObserver issue but bigger picture I think we need to create a ticket and assign a person responsible for fixing or silencing that error in DD (I feel like we circled around this topic a few times already but never got to action)

Reply:
Yes, I think we need to create a ticket and see if it can be picked up next sprint. Not a blocker here, but just make sure this addition properly unmounts, prevents loops, etc.

Logging in the callback(s) and playing around in the sandbox can likely help identify some initial issues

Reply:
Sgtm, will bring it up in the standup parking lot for discussion 👍